invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/administration/users/UserActions.js

```diff
@@ -101,11 +101,18 @@ export class UserActions extends Component {
       displayImpersonateUser,
       displayQuota,
       useDropdown,
+      recordId,
     } = this.props;
     const { loading } = this.state;
     const isUserBlocked = !isEmpty(user.blocked_at);
     const isUserActive = user.active;
     const isUserVerified = !isEmpty(user.verified_at);
+    const apiUrl = recordId
+      ? `/api/records/${recordId}/quota`
```

Comment:
Since the component is tied to the user, it should not be aware of a record.
I think this is the moment to split the functionalities and either
1) have a separate very similar component `RecordActions` where we implement `<SetQuotaAction` and `<UserActions` returned in render function
OR: 2) we take out the `<Set quota` from this component and we place it directly in respective SearchResultItemLayout or users and record

I would prefer to avoid adding more responsibilities to `UserActions` since we aim to have single responsibility components and it is already quite overloaded